packages/react-aria-components/stories/ListBox.stories.tsx

```diff
@@ -181,6 +181,70 @@ ListBoxDnd.story = {
   }
 };
 
+export const ListBoxDndCustomDropIndicator = (props: ListBoxProps<typeof albums[0]>) => {
```

Comment:
Can we make this story a little nicer, each image takes up so much of my screen that it's hard to use
it'd be really cool if we could use the image in the custom drop indicator as well, I feel like that'd be more true to life/the use case that was asked for. Wonder how hard it'd be to add the animations as well.